### DISCUSSION TITLE
How to Tag someone in a Group?

### DISCUSSION THREAD

**Discussion by azickri** [tulir/whatsmeow#562] [General]
![image](https://github.com/tulir/whatsmeow/assets/45684749/f7a7e182-e4bd-45d8-ab66-bd2e03e394a5)

Hello, Is whatsmeow supported for doing this?
If so, can you provide a documentation link? Because I didn't find it in the Documentation.

**Comment by ghost** [comment]
It is called mentioning 

already there : https://github.com/tulir/whatsmeow/discussions/259, 

```go
// Example ContextInfo for ExtendedTextMessage

var msg = &waProto.Message{
    ExtendedTextMessage: &waProto.ExtendedTextMessage{
      Text:        proto.String("Mentioning @1234567XXXX "),
      ContextInfo: &waProto.ContextInfo{
        MentionedJid: []string{
          event.Info.Sender.ToNonAD().String(),
        },
      },
    },
  }
```

**Reply by azickri** [reply]
Thanks, I found it a while ago. Forgot to explain again in this discussion.
